frappe/public/js/frappe/form/toolbar.js

```diff
@@ -52,10 +52,14 @@ frappe.ui.form.Toolbar = Class.extend({
 		this.set_indicator();
 	},
 	is_title_editable: function() {
-		if (this.frm.meta.title_field==="title"
+		let title_field = this.frm.meta.title_field;
+		let field = this.frm.get_field(title_field);
+		if (title_field==="title"
```

Comment:
true, but that part is unchanged since 4 years 
should this be changed to

```javascript
if (title_field
	&& this.frm.perm[0].write
	&& !this.frm.get_docfield("title").options
	&& !this.frm.doc.__islocal
	&& field
	&& !field.df.read_only) {
	return true;
} else {
	return false;
}
```

Reply:
I set the `title_field` to `status` from Customize Form in Project.

And the rename doesn't work:
![image](https://user-images.githubusercontent.com/9355208/66451604-d01d4680-ea7a-11e9-8d65-fdbb998a98ef.png)